test/fail_compilation/misc1.d

```diff
@@ -0,0 +1,20 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/misc1.d(108): Error: `5` has no effect
+fail_compilation/misc1.d(109): Error: `1 + 2` has no effect
+---
+*/
+
+#line 100
```

Comment:
FYI: I you wander about the name, I plan to reduce the count of files of the "fail" test suite by merging individuals tests, and if they dont require args, by merginf in this file.